PsfLauncher/main.cpp

```diff
@@ -268,43 +275,247 @@ void LaunchMonitorInBackground(std::filesystem::path packageRoot, const wchar_t
         }
         else
         {
-            //Log("error starting monitor using SellExecuteEx also. Error=0x%x\n", ::GetLastError());
+            std::wstring errorString = L"error starting monitor using ShellExecuteEx";
+            auto err = ::GetLastError();
+            ErrorInformation error = { errorString, err, executable };
+

```

Comment:
why all the locals?  this could be nearly one line

Reply:
I changed it one line with return {...}

But, the answer to why is because I didn't know you could do it in one line.